src/ensembl/src/content/app/custom-download/containers/content/result-holder/ResultHolder.tsx

```diff
@@ -6,10 +6,10 @@ import {
   getIsLoadingResult
 } from '../../../state/customDownloadSelectors';
 
-import { getAttributes } from '../attributes-accordion/state/attributesAccordionSelector';
-import { getFilters } from '../filter-accordion/state/filterAccordionSelector';
+import { getSelectedAttributes } from '../attributes-accordion/state/attributesAccordionSelector';
+import { getSelectedFilters } from '../filter-accordion/state/filterAccordionSelector';
 
-import AttributesSection from 'src/content/app/custom-download/types/Attributes';
+import { keys } from 'lodash';
```

Comment:
This imports the whole lodash library!

Reply:
It's not needed. Just to confirm, do we use loadash only when there are some compatibility issues?

Reply:
> Just to confirm, do we use loadash only when there are some compatibility issues?
Primarily when it offers us some convenience. The `find` vs `Array.prototype.find` question is still something I don't have a strong opinion on. Perhaps we should use the native `Array.prototype.find` and add a polyfill for browsers that do not support it (that will be only IE11).

Reply:
In this case, should we say that we can only use certain functions?

Reply:
Just wanted to add my two cents here. We (@msmanoj and I) had a chat about this in the morning. The native `find` is a lot faster than the lodash `find`: https://measurethat.net/Benchmarks/ShowResult/63691.

Reply:
@imran-ebi @msmanoj I am ok if we agree to use Array.prototype.find instead of lodash's `find`. Lodash's `find` is allowed if ever we decide to use functions from `lodash/fp` (which is like ramda, only lodash 😄 )

I think, the general guideline is, prefer native functions over lodash's functions if native function do exactly the same job and there is no apparent benefit to using lodash's equivalents.